lesson10/index.md

```diff
@@ -120,9 +120,9 @@ public static void main(String[] args) {
 }
 ```
 
-A **static** variable or a **static** method can accessed without having an instance of the class. This methods and variable are even existent, if no instance of the class exists at all. But a static variable of a class with the same name exists just once. So if a static variable is modified from somewhere in the program this modification is valid for the following times this variable is accessed until its value is changed again. 
+A **static** variable or a **static** method can be accessed without having an instance of the class. These methods and variable are even NOT existent, if no instance of the class exists at all. But a static variable of a class with the same name exists just once. So if a static variable is modified from somewhere in the program this modification is valid for the following times this variable is accessed until its value is changed again. 
```

Comment:
the NOT is not needed,
that part was actually correct ... static methods/variables are existent if no instance of class exists

Reply:
Alright, thanks! I'd like to draw it back and thanks for the explantion!